examples/using_fhir_views.ipynb

```diff
@@ -0,0 +1,671 @@
+      "Requirement already satisfied: google-fhir-r4~=0.9.2 in /usr/local/google/home/omarismail/.local/lib/python3.10/site-packages (from google-fhir-views[r4,spark]) (0.9.2)\n",
+      "Requirement already satisfied: stringcase==1.2.0 in /usr/local/google/home/omarismail/.local/lib/python3.10/site-packages (from google-fhir-core~=0.9.2->google-fhir-views[r4,spark]) (1.2.0)\n",
+      "Requirement already satisfied: antlr4-python3-runtime~=4.9.3 in /usr/local/google/home/omarismail/.local/lib/python3.10/site-packages (from google-fhir-core~=0.9.2->google-fhir-views[r4,spark]) (4.9.3)\n",
+      "Requirement already satisfied: numpy>=1.21.0 in /usr/lib/python3/dist-packages (from pandas~=1.1->google-fhir-views[r4,spark]) (1.24.2)\n",
+      "Requirement already satisfied: pytz>=2020.1 in /usr/lib/python3/dist-packages (from pandas~=1.1->google-fhir-views[r4,spark]) (2022.7.1)\n",
+      "Requirement already satisfied: six>=1.5 in /usr/lib/python3/dist-packages (from python-dateutil~=2.8->google-fhir-views[r4,spark]) (1.16.0)\n"
+     ]
+    }
+   ],
+   "source": [
+    "!pip3 install google-fhir-views[r4,spark]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas\n",
+    "\n",
+    "from sqlalchemy import dialects\n",
+    "from sqlalchemy import engine\n",
+    "\n",
+    "from google.fhir.views import r4\n",
+    "from google.fhir.views import spark_runner\n",
+    "\n",
+    "# The Spark dataset containing FHIR data. This may be read-only to the user.\n",
+    "fhir_dataset = \"default\"\n",
+    "\n",
+    "# The Spark dataset where we will create views, value sets, and other derived tables\n",
+    "# as needed. This must be writeable by the user. This will use the default project\n",
+    "# where this notebook is running.\n",
+    "analysis_dataset = \"statin_analysis_example\"\n",
+    "\n",
+    "dialects.registry.register(\"hive\", \"pyhive.sqlalchemy_hive\", \"HiveDialect\")\n",
+    "\n",
+    "# The endpoint of the Hive ThriftServer to connect to\n",
+    "query_engine = engine.create_engine(\"hive://localhost:10001/default\")\n",
+    "\n",
+    "# Create a runner to execute the views over Spark.\n",
+    "runner = spark_runner.SparkRunner(\n",
+    "    query_engine=query_engine,\n",
+    "    fhir_dataset=fhir_dataset,\n",
+    "    view_dataset=analysis_dataset,\n",
+    "    snake_case_resource_tables=True,\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The cell below loads the parquet files we've downloaded into our Spark ThriftServer:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "destination_directory_path = \"/dwh/controller_*\"\n",
+    "with query_engine.connect() as curs:\n",
```

Comment:
Just FYI, these tables are automatically created if `createHiveResourceTables` is set like [here](https://github.com/google/fhir-data-pipes/blob/3694f1394d0b9011ab480ca61f0bd0568bca2f53/docker/config/application.yaml#L34). It is okay to include this cell to make it clear how people can (re)create these independently but maybe add a reference to that setting too.